src/Components/Home/Searchbar.css

```diff
@@ -0,0 +1,3 @@
+.search-bar-resize {
+  width: 180px;
+}
```

Comment:
Is a own file for this really needed or could we add this in the main style sheet?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
Good question. I'm open to suggestions on which way is best.

Reply:
I would suggest using the main style sheet due the modifications for the Search-Bar is minor. If it would be a big change I would prefer the own stylesheet but this is only personal preference. Maybe some other @EddieHubCommunity/linkfree maintainer could share some thoughts on this topic.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Elegant, Maintainable